bin/src/system_tray.rs

```diff
@@ -25,19 +20,6 @@ pub(crate) fn build(app: &AppHandle) -> AppResult<()> {
         .icon(app.default_window_icon().unwrap().clone())
         .menu(&menu)
         .on_menu_event(event_handler)
-        .on_tray_icon_event(|tray, event| {
-            if let TrayIconEvent::Click {
-                button: MouseButton::Left,
-                ..
-            } = event
-            {
-                let app = tray.app_handle();
-                if let Some(window) = app.get_webview_window("main") {
-                    let _ = window.show();
-                    let _ = window.set_focus();
-                }
-            }
-        })
```

Comment:
@naps62 don't remember if we finished our conversation about why this was here but make sure to check if these changes don't break in Linux

Reply:
don't think we discussed this much, yeah
after trying your PR, I'm inclined to go in the other direction: remove the "Show" button, and let the default behavior of the icon open the window (seems to work at least on Hyprland, so I assume it's expected)